src/app/pages/TopicPage/Pagination/index.jsx

```diff
@@ -191,29 +191,44 @@ const renderBlock = ({
 };
 /* eslint-enable react/prop-types */
 
+const getTranslations = translations => ({
+  pageXOfY: 'Page {x} of {y}',
+  previousPage: 'Previous Page',
+  nextPage: 'Next Page',
+  ...translations.pagination,
+});
+
 const Pagination = ({ activePage, pageCount }) => {
-  const { service } = useContext(ServiceContext);
+  const { service, translations } = useContext(ServiceContext);
   const blocks = buildBlocks(activePage, pageCount);
   if (!blocks) return null;
 
+  const { pageXOfY, previousPage, nextPage } = getTranslations(translations);
+  const summaryText = pageXOfY
+    .replace('{x}', `<b>${activePage}</b>`)
+    .replace('{y}', `<b>${pageCount}</b>`);
```

Comment:
Seeking advice on a better way of doing this - can't think of anything that doesn't require either `dangerouslySetInnerHTML`, a mountain of code or a dependency